sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -429,6 +429,12 @@ def begin_recognize_content(self, form, **kwargs):
         :keyword list[str] pages: Custom page numbers for multi-page documents(PDF/TIFF), input the number of the
             pages you want to get OCR result. For a range of pages, use a hyphen. Separate each page or
             range with a comma.
+        :keyword str language: The BCP-47 language code of the text in the document. Currently, only English
```

Comment:
can you make this docstring more like `locale` with just the shorter list of the language codes? https://github.com/Azure/azure-sdk-for-python/blob/9928714133fdb95d44fb9c1293eeba5758ba6ec4/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py#L93